website/source/docs/platform/k8s/mesh-gateways.html.md

```diff
@@ -0,0 +1,360 @@
+
+# You should also have the config specified in the prerequisites.
+```
+
+And then upgrade your chart:
+
+```bash
+# Get your chart name.
+helm list
+
+# Then upgrade it.
+helm upgrade <your release name> . -f dc1-values.yaml
+```
+
+You should see the gateway pods starting:
+
+```bash
+$ kubectl get pod -l component=mesh-gateway
+sanguine-elk-consul-mesh-gateway-5c9475b97f-h2g9g                 1/1     Running   0          11m
+sanguine-elk-consul-mesh-gateway-5c9475b97f-ptkbd                 1/1     Running   0          11m
+```
+
+### Step 3 - Wait for the Load Balancer IP
+We're going to use the load balancer's IP to route to the gateways so we need
+to wait for that to be up.
+
+Run
+
+```bash
+kubectl get svc -l component=mesh-gateway
+```
+and wait until you see an external IP:
+
+```bash
+NAME                               TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)         AGE
+sanguine-elk-consul-mesh-gateway   LoadBalancer   10.35.254.171   32.32.32.32     443:32022/TCP   8m6s
+```
+
+Now change the `meshGateway.wanAddress.host` key to the external IP.
+This will tell remote datacenters to route gateway traffic to the load balancer.
+
+```diff
+meshGateway:
+  enabled: true
+  wanAddress:
+-    useNodeIP: true
++    useNodeIP: false
++    host: 32.32.32.32
+```
+
+Upgrade your helm release again:
+
+```bash
+helm upgrade <your release name> . -f dc1-values.yaml
+```
+
+### Step 4 - Configure Your Second DC
+Now we're ready to start Consul in our second DC (`dc2`).
+
+Change your `kubectl` context:
+
+```bash
+kubectl config get-contexts
+...
+kubectl config use-context your-dc2-context
+```
+
+Add the following config to your helm chart values for dc2.
+
+```yaml
+# dc2-values.yaml
+meshGateway:
+  enabled: true
+  wanAddress:
+    useNodeIP: true
+  enableHealthChecks: false
+  service:
+    enabled: true
+    type: LoadBalancer
+
+# You should also have datacenter config:
+global:
+  datacenter: dc2
+    
+# You should also have the config specified in the prerequisites.
+```
+
+And then upgrade your chart:
+```bash
+helm upgrade <your release name> . -f dc2-values.yaml
+```
+
+### Step 5 - Wait for the Load Balancer IP
+We're going to use the load balancer's IP to route to the gateways so we need
+to wait for that to be up.
+
+Run
+
+```bash
+kubectl get svc -l component=mesh-gateway
+```
+and wait until you see an external IP:
+
+```bash
+NAME                               TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)         AGE
+sanguine-elk-consul-mesh-gateway   LoadBalancer   10.35.254.171   32.32.32.32     443:32022/TCP   8m6s
+```
+
+Now change the `meshGateway.wanAddress.host` key to the IP:
+
+```diff
+meshGateway:
+  enabled: true
+  wanAddress:
+-    useNodeIP: true
++    useNodeIP: false
++    host: 32.32.32.32
+```
+
+Upgrade your helm release again:
+
+```bash
+helm upgrade <your release name> . -f dc2-values.yaml
+```
+
+### Step 6 - Ensure each datacenter can access the Load Balancers
```

Comment:
Is this correct? I think the idea was to have the gateways talk to each other so that you didn't have to expose sidecar proxy pods to the outside world directly.

Reply:
Yes you're totally correct! Good catch, I'll rework this :D